bats/tests/snapshots/test-snapshot-list.bats

```diff
@@ -26,15 +26,31 @@ local_setup() {
     assert_output 'No snapshots present.'
 }
 
-@test 'verify snapshot-list output with snapshots' {
+@test 'create three snapshots, allowing for restarts' {
+    # Ensure the app is up and running
+    wait_for_container_engine
+    wait_for_apiserver
+    wait_for_backend
```

Comment:
Is it necessary to have all three of these? It seems that it would be sufficient to have only `wait_for_backend` - if you think about it, this is what `rdctl snapshot create/restore` rely on, and that works fine.

As a sidenote, there is a lot of repetition here - would a for loop make sense?

Reply:
I want all three there because one of them could time out, and it would be helpful to know which one we froze at.

I avoided the repetition because I want at least 5 seconds after creating the first two snapshots, but not the third, and the extra logic seemed to outweigh the benefits of rolling 3 instances into a loop.